scene/2d/camera_2d.cpp

```diff
@@ -141,6 +141,14 @@ Transform2D Camera2D::get_camera_transform() {
 				camera_pos.y -= screen_rect.position.y - limit[MARGIN_TOP];
 		}
 
+		if (snapping_enabled && !Engine::get_singleton()->is_editor_hint()) {
```

Comment:
I would usually surround the `is_editor_hint()` check around `#ifdef TOOLS_ENABLED`.

Reply:
No need anymore, it's defines like this:
```
#ifdef TOOLS_ENABLED
        _FORCE_INLINE_ void set_editor_hint(bool p_enabled) { editor_hint = p_enabled; }
        _FORCE_INLINE_ bool is_editor_hint() const { return editor_hint; }
#else
        _FORCE_INLINE_ void set_editor_hint(bool p_enabled) {}
        _FORCE_INLINE_ bool is_editor_hint() const { return false; }
#endif
```

Reply:
I think it won't work because it has to work in game but not in the editor. Anyways I did it the same way as `smoothing_enabled` and other related properties.